var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -212,6 +217,18 @@ def cmake_args(self):
                 '-DDEAL_II_DOCHTML_RELDIR=share/deal.II/doc'
             ])
 
+        # CUDA
+        # FIXME  -DDEAL_II_CUDA_FLAGS="-arch=sm_60"
```

Comment:
@tgamblin i presume so far there is nothing in Spack which would help to know the CUDA device architecture?